libs/gi/page-team/src/TeamCharacterSelector.tsx

```diff
@@ -103,9 +127,50 @@ export default function TeamCharacterSelector({
         }
       >
         <CardContent sx={{ display: 'flex', justifyContent: 'center', pb: 0 }}>
-          <Typography variant="h5" display="flex">
-            {team.name}
-          </Typography>
+          <CardActionArea>
+            <Typography
+              variant="h5"
+              align="center"
+              onClick={() => setEditMode(true)}
+            >
+              {teamName}
+            </Typography>
```

Comment:
Something for later after some feedback probably, but we might need to add an edit icon here. And maybe only allow editing/show the edit icon on the team settings page so it is more obvious